agglayer/types/types.go

```diff
@@ -1338,3 +1338,39 @@ type ClockConfiguration struct {
 func (c ClockConfiguration) String() string {
 	return fmt.Sprintf("EpochDuration: %d, GenesisBlock: %d", c.EpochDuration, c.GenesisBlock)
 }
+
+// NetworkStatus represents the status of the network returned by the interop_getNetworkStatus RPC call
+type NetworkStatus struct {
+	// NetworkStatus is the current status of the network (e.g., "active", "syncing", "error")
+	NetworkStatus string `json:"network_status"`
```

Comment:
Just thinking outloud, rename it to `State` or `Status`, in order to not have the same name as the struct, WDYT?

```suggestion
	Status string `json:"network_status"`
```

Reply:
I am naming them as the proto struct, but I do agree it might be more readable. I will rename it.

Reply:
Fix: https://github.com/agglayer/aggkit/pull/913/commits/88047617be5bec9b62816b8fddf43a1056601e99